Source/Editor/Surface/Archetypes/Parameters.cs

```diff
@@ -937,13 +952,25 @@ public void OnParamDeleted(SurfaceParameter param)
             {
                 // Deselect if that parameter is selected
                 if ((Guid)Values[0] == param.ID)
-                {
                     _combobox.SelectedIndex = -1;
-                }
 
                 UpdateCombo();
             }
 
+            /// <inheritdoc />
+            public bool IsParamreferenced(SurfaceParameter param, VisjectSurface surface)
+            {
+                for (int i = 0; i < surface.Nodes.Count; i++)
```

Comment:
Simplify this code into `return (Guid)Values[0] == param.ID;`.

Reply:
Added in https://github.com/FlaxEngine/FlaxEngine/pull/3010/commits/b055117144d42df15435a98207e41299161185c6. Thanks for the review